backend/config/accounts.go

```diff
@@ -42,7 +42,9 @@ type Account struct {
 	// If false, the account is only displayed if the keystore is connected. If true, it is loaded
 	// and displayed when the app launches.
 	//
-	// If nil, it is considered false.
+	// If nil, it is considered false.  The reason for this is that we don't want to suddenly show
+	// all persisted accounts when the Watchonly setting is enabled - only accounts that are loaded
+	// when Watchonly is enabled should do this.
```

Comment:
i actually spent a lot of hours at the beginning deleting this and putting in a var named exactly like you suggest. in the very end it turned out that everything is the same except that `HideFromWatchonly` is the negation of `Watch`, so not really any simpler. then i reverted that part and left `Watch` so that the diff of this PR is smaller and clearer. :shrug:

Reply:
lol ok :joy: